soc/arm/nordic_nrf/nrf51/power.c

```diff
@@ -1,11 +1,96 @@
 /*
- * Copyright (c) 2018 Intel Corporation.
+ * Copyright (c) 2017 Intel Corporation.
  *
  * SPDX-License-Identifier: Apache-2.0
  */
 #include <zephyr.h>
+#include <soc_power.h>
 #include <nrf_power.h>
 
+#define LOG_LEVEL CONFIG_SOC_LOG_LEVEL
+#include <logging/log.h>
+LOG_MODULE_DECLARE(soc);
```

Comment:
use `LOG_MODULE_DECLARE(soc, CONFIG_SOC_LOG_LEVEL);` and remove `#define LOG_LEVEL ...`

Reply:
Fixed for nRF51 and nRF52. Thank you.